frontend/src/scenes/toolbar-launch/AuthorizedUrlsTable.tsx

```diff
@@ -0,0 +1,178 @@
+                        {record.type === 'suggestion' ? (
+                            <LemonButton type="default" onClick={() => addUrl(record.url)}>
+                                <PlusOutlined /> Add as authorized
+                            </LemonButton>
+                        ) : (
+                            <>
+                                <a href={launchUrl(record.url)}>
+                                    <LemonButton type="highlighted">Launch toolbar</LemonButton>
+                                </a>
+                                <Popup
+                                    visible={popoverOpen === `${index}_${record.url}`}
+                                    actionable
+                                    onClickOutside={() => setPopoverOpen(null)}
+                                    onClickInside={() => setPopoverOpen(null)}
+                                    overlay={
+                                        <>
+                                            <LemonButton
+                                                fullWidth
+                                                type="stealth"
+                                                onClick={() => setEditUrlIndex(record.originalIndex)}
+                                            >
+                                                <EditOutlined style={{ marginRight: 4 }} />
+                                                Edit authorized URL
+                                            </LemonButton>
+                                            <LemonButton
+                                                fullWidth
+                                                style={{ color: 'var(--danger)' }}
+                                                type="stealth"
+                                                onClick={() => removeUrl(index)}
+                                            >
+                                                <DeleteOutlined style={{ marginRight: 4 }} />
+                                                Remove authorized URL
+                                            </LemonButton>
+                                        </>
+                                    }
+                                >
+                                    <LemonButton
+                                        type="stealth"
+                                        onClick={() => setPopoverOpen(popoverOpen ? null : `${index}_${record.url}`)}
+                                        style={{ marginLeft: 8 }}
+                                    >
+                                        <EllipsisOutlined
+                                            style={{ color: 'var(--primary)', fontSize: 24 }}
+                                            className="urls-dropdown-actions"
+                                        />
+                                    </LemonButton>
+                                </Popup>
```

Comment:
The standard `<More/>` component here that's used in all other lemon tables should be preferred here